### PR TITLE
Issue #419: Only add Tier-1 and Tier-2 builders to Release Status

### PR DESCRIPTION
**Pull request by vstinner**

### CODE REVIEW

**Comment by vstinner** [comment]
cc @ambv @pablogsal @zware

File: master/master.cfg [hunk 1]
```diff
@@ -252,6 +253,9 @@ for branch_num, (git_url, branchname, git_branch) in enumerate(git_branches):
             # disable GitHub notifications for unstable builders
             if stability == STABLE:
                 github_status_builders.append(buildername)
+                # Only Tier-1 and Tier-2 builders can block a release
+                if tier in (TIER_1, TIER_2):
```
Comment (ambv) [inline master/master.cfg:257]: builtins.NameError: name 'TIER_1' is not defined

Comment (vstinner) [reply master/master.cfg:258]: Fixed ;-)

**Comment by vstinner** [comment]
cc @pablogsal @Yhg1s @ambv: I fixed https://buildbot.python.org/all/#/release_status page to only list buildbot builders of Tier-1 and Tier-2. Builders of Tier-3 or with no Tier must not block a release and so are no longer listed.

**Pull request merged**